src/Raven.Server/Documents/Handlers/QueriesHandler.cs

```diff
@@ -333,6 +339,21 @@ public Task Patch()
                 throw;
             }
         }
+        /// <summary>
+        /// TrafficWatchQuery writes query data to httpContext
+        /// </summary>
+        /// <param name="indexQuery"></param>
+        private void TrafficWatchQuery(IndexQueryServerSide indexQuery)
+        {
+            var sb = new StringBuilder();
+            // append stringBuilder with the query
+            sb.Append(/*"Query:\n"+ */indexQuery.Query);
+            // if query got parameters append with parameters
+            if (indexQuery.QueryParameters != null && indexQuery.QueryParameters.Count > 0)
+                sb.Append("\n" + indexQuery.QueryParameters);
```

Comment:
This will _not_ generate good output.

Reply:
This is the output generated

![image](https://user-images.githubusercontent.com/16625387/45596676-41014480-b9c8-11e8-9c69-0a3f20bfe280.png)

Reply:
How, `QueryParameters` should just return the type name